vcl_templates/www.vcl.erb

```diff
@@ -220,6 +220,10 @@ sub vcl_recv {
     set req.http.GOVUK-ABTest-EducationNavigation = "A";
   } else if (req.url ~ "[\?\&]ABTest-EducationNavigation=B(&|$)") {
     set req.http.GOVUK-ABTest-EducationNavigation = "B";
+  } else if (req.url ~ "^\/education(\/)?.*$") {
```

Comment:
Are there any content pages at URLs like www.gov.uk/education-and-some-other-info which would be affected by that?